target/linux/ramips/image/mt7620.mk

```diff
@@ -37,6 +37,10 @@ define Build/elecom-header
 	$(STAGING_DIR_HOST)/bin/tar -cf $@ -C $(KDIR) v_0.0.0.bin v_0.0.0.md5
 endef
 
+define Build/zyimage
+	zyimage $(1) $@
```

Comment:
$(STAGING_DIR_HOST)/bin/zyimage $(1) $@

Otherwise it might runs a zyimage from you system instead the LEDE one.